tests/core/data/test_tensor_train.py

```diff
@@ -32,6 +32,23 @@ def assert_bond_edges_name(tt):
         assert tt.train_nodes[i + 1]["lbond"] is edge
 
 
+def random_mpo(n, d, bond_dimension):
+    """Create a random mpo with n sites d dimension per site and
+    bond_dimesnion."""
+    if n > 1:
+        list_tensors = [np.random.random((d, d, bond_dimension)) - 1 / 2]
```

Comment:
I wonder if we should add some imaginary components here to make these tensors complex so that we can test that conjugation happens correctly?